python/datasources/cdc_restricted_local.py

```diff
@@ -0,0 +1,251 @@
+    COUNTY_FIPS_COL: std_col.COUNTY_FIPS_COL,
+    COUNTY_COL: std_col.COUNTY_NAME_COL,
+    RACE_COL: std_col.RACE_OR_HISPANIC_COL,
+    SEX_COL: std_col.SEX_COL,
+    AGE_COL: std_col.AGE_COL,
+}
+
+# Mapping for county_fips, county, and state unknown values to "Unknown".
+COUNTY_FIPS_VALUES_MAPPING = {"NA": "-1"}  # Has to be str for later ingestion.
+COUNTY_VALUES_MAPPING = {"MISSING": "Unknown", "NA": "Unknown"}
+STATE_VALUES_MAPPING = {"Missing": "Unknown", "NA": "Unknown"}
+
+# Mappings for race, sex, and age values in the data to a standardized forms.
+# Note that these mappings cover the possible values in the data as of 3/1/21.
+# New data should be checked for schema changes.
+RACE_VALUES_MAPPING = {
+    "American Indian/Alaska Native, Non-Hispanic": std_col.Race.AIAN_NH.value,
+    "Asian, Non-Hispanic": std_col.Race.ASIAN_NH.value,
+    "Black, Non-Hispanic": std_col.Race.BLACK_NH.value,
+    "Multiple/Other, Non-Hispanic": std_col.Race.MULTI_NH.value,
+    "Native Hawaiian/Other Pacific Islander, Non-Hispanic": std_col.Race.NHPI_NH.value,
+    "White, Non-Hispanic": std_col.Race.WHITE_NH.value,
+    "Hispanic/Latino": std_col.Race.HISP.value,
+    "NA": std_col.Race.UNKNOWN.value,
+    "Missing": std_col.Race.UNKNOWN.value,
+    "Unknown": std_col.Race.UNKNOWN.value,
+}
+
+SEX_VALUES_MAPPING = {
+    "NA": "Unknown",
+    "Missing": "Unknown",
+}
+
+AGE_VALUES_MAPPING = {
+    "0 - 9 Years": "0-9",
+    "10 - 19 Years": "10-19",
+    "20 - 29 Years": "20-29",
+    "30 - 39 Years": "30-39",
+    "40 - 49 Years": "40-49",
+    "50 - 59 Years": "50-59",
+    "60 - 69 Years": "60-69",
+    "70 - 79 Years": "70-79",
+    "80+ Years": "80+",
+    "NA": "Unknown",
+    "Missing": "Unknown",
+}
+
+
+def accumulate_data(df, groupby_cols, overall_df, demog_col,
+                    values_mapping):
+    """Converts/adds columns for cases, hospitalizations, deaths. Does some
+    basic standardization of dataframe elements. Groups by given groupby_cols
+    and aggregates. Returns sum of the aggregated df & overall_df.
+
+    df: Pandas dataframe that contains a chunk of all of the raw data.
+    groupby_cols: List of columns we want to groupby / aggregate on.
+    overall_df: Pandas dataframe to add our aggregated data to.
+    demog_col: Name of the demographic column to standardize.
+    values_mapping: Mapping from demog value to standardized form.
+    """
+    # Add a columns of all ones, for counting the # of cases / records.
+    df[std_col.COVID_CASES] = np.ones(df.shape[0], dtype=int)
+
+    # Add columns for hospitalization yes/no/unknown and death yes/no/unknown,
+    # as we aggregate and count these individually. Do a sanity check that we
+    # covered all the data and drop the original hospitalization/death columns.
+    df[std_col.COVID_HOSP_Y] = (df['hosp_yn'] == 'Yes')
+    df[std_col.COVID_HOSP_N] = (df['hosp_yn'] == 'No')
+    df[std_col.COVID_HOSP_UNKNOWN] = ((df['hosp_yn'] == 'Unknown') |
```

Comment:
Might better logic just be != 'Yes and != 'No'.  Makes it more future proof.  For instance if they change 'hosp_yn' to 'NA'